pom.xml

```diff
@@ -6,12 +6,113 @@
 
     <groupId>com.epam.tc</groupId>
     <artifactId>AlenaSidorova</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>2021.08.03-SNAPSHOT</version>
 
     <properties>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <testng.configs.path>${project.build.testOutputDirectory}/testng</testng.configs.path>
+        <testng.suite.file>${testng.configs.path}/test.AddAndSubtractOperations.xml</testng.suite.file>
+
+        <checkstyle.version>8.44</checkstyle.version>
```

Comment:
As far as I remember, our checkstyle configuration requires 8.42
Could you please recheck it?

Reply:
Maybe I've misunderstood something, but I haven't found any version requirements for this plugin in Homework file, so I decided to put the latest version, which was suggested by IntelliJ IDE. The latest version is 8.45 for the moment (https://checkstyle.org/). 
Please let me know - what version should I use in a project?  I'll change it.

Reply:
Saying "Classes have something in common, maybe put them in hierarchy could be a good solution" what exactly do you mean? To make an abstract test class? If it's true maybe we could also put the @Before method (suggested by you in a comment above) right  there?

Reply:
An example of checkstyle maven plugin usage has `8.42` https://github.com/DmitryKhodakovsky/epam-training-center-code-checkers-configurations/blob/main/pom.xml#L16  sometimes this can be critical. Please use 8.42

I mean create some parent class for test classes that holds the calculator and initialize it in BeforeSomething method of this class